frontend/packages/kubevirt-plugin/integration-tests/tests/cnv-installation.scenario.ts

```diff
@@ -0,0 +1,25 @@
+import * as sidenavView from '@console/internal-integration-tests/views/sidenav.view';
+import { isLoaded } from '@console/internal-integration-tests/views/crud.view';
+import { $, browser, by, element } from 'protractor';
+import { click } from '@console/shared/src/test-utils/utils';
+
+const namespaceButton = $('.co-namespace-selector button');
+const openshiftNamespaceButton = $('#openshift-cnv-link');
+
+ 
+describe('Go to operators page', () => {
+ 
+  beforeAll(async () => {
+    await sidenavView.clickNavLink(['Operators', 'OperatorHub']);
+    await isLoaded();
+    await click(namespaceButton);
+    await click(openshiftNamespaceButton);
+  });
+ 
+  it('install Kubevirt', async () => {
+    await isLoaded();
+    element(by.cssContainingText('.catalog-tile-pf-title', 'Container-native virtualization Operator')).click();
+    element(by.linkText('Install')).click()
+    await browser.sleep(20000);  
```

Comment:
We should probably wait asynchronously for installation suceeded. This implementation would pass even when the installation failed. Storage team has already worked on this, maybe you could use the code they implemented:
https://github.com/openshift/console/blob/28cfe3b2fc4b233011cb0145a7acb5ecba6876c7/frontend/packages/ceph-storage-plugin/integration-tests/utils/helpers.ts
So instead of the `await browser.sleep(20000);` you could use:
`await waitFor(kubevirtOperatorStatus, 'Succeeded', 5);`
where kubevirtOperatorStatus could probably be `$('.co-clusterserviceversion-row__status');`

Reply:
Additionally, we should pass custom timeout to the `it` method, this test case may take more than 40 seconds (which is default)